utility-belt/src/test/java/io/confluent/admin/utils/EmbeddedZookeeperEnsemble.java

```diff
@@ -54,6 +54,7 @@
   private int tickTime = 2000;
   private int initLimit = 3;
   private int syncLimit = 3;
+  private int connectToLearnerMasterLimit = 3;
```

Comment:
```suggestion
  private int connectToLearnerMasterLimit = initLimit;
```

Reply:
Yeah, that was what I was talking about.